docs/software/home_automation.md

```diff
@@ -221,4 +222,61 @@ Official website: <https://www.openhab.org/>
 Official docs: <https://www.openhab.org/docs/>  
 Official community: <https://community.openhab.org/>
 
+## Homebridge
+
+Homebridge allows you to integrate with smart home devices that do not natively support HomeKit. There are over 2,000 Homebridge plugins supporting thousands of different smart accessories.
+
+![Homebridge logo](../assets/images/dietpi-software-homeautomation-homebridge.svg){: width="300" height="300" loading="lazy"}
+
+=== "Access to the web interface"
+
+    The web interface is accessible via port **8581**:
+
+    - URL: `https://<your.IP>:8581`
+    - Username/Password: Are set at first web interface login
+
+=== "Directories"
+
+    - Install directory: `/usr/local/bin/hb-service`
```

Comment:
This is the executable symlink. What comes closest to the install directory is `/opt/homebridge`, which also contains the actual binary.

Reply:
Fixed in https://github.com/MichaIng/DietPi-Docs/pull/904/commits/d7930000fca34c4b85241207c9496fc29ef68ec2